bin/alert_on_unused_contexts.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from services.github_service import GithubService
+from services.circleci_service import CircleciService
+from services.slack_service import SlackService
+
+# pylint: disable=W0718, R0914
+
+GITHUB_ORG = "ministryofjustice"
+
+
+def get_environment_variables() -> tuple:
+    slack_token = os.getenv("ADMIN_SLACK_TOKEN")
+    if not slack_token:
+        raise ValueError(
+            "The env variable ADMIN_SLACK_TOKEN is empty or missing")
+    github_token = os.getenv("ADMIN_GITHUB_TOKEN")
+    if not github_token:
+        raise ValueError(
+            "The env variable ADMIN_GITHUB_TOKEN is empty or missing")
+    circleci_token = os.getenv("ADMIN_CIRCLECI_TOKEN")
+    if not circleci_token:
+        raise ValueError(
+            "The env variable ADMIN_CIRCLECI_TOKEN is empty or missing")
+    circle_owner_id = os.getenv("CIRCLE_CI_OWNER_ID")
```

Comment:
perhaps this should be called `circleci_owner_id` just to be in keeping with the other refs to CircleCI